api/views.py

```diff
@@ -0,0 +1,47 @@
+from rest_framework.response import Response
+from rest_framework.viewsets import ModelViewSet
+from rest_framework import status
+
+from . models import Category, Job, Application
+from . serializers import JobSerializer, ApplicationSerializer
+from . common.constants import DATA_SAVED_SUCCESSFULLY, INVALID_DATA, DATA_ALREADY_EXISTS
```

Comment:
@amina-cg avoid these kinds of relative imports.
```suggestion
from api.common.constants import DATA_SAVED_SUCCESSFULLY, INVALID_DATA, DATA_ALREADY_EXISTS
```

Reply:
Do I need to implement it on models, serializers as well?

Reply:
yes

Reply:
Updated.